data/core/commandview.lua

```diff
@@ -61,6 +62,7 @@ function CommandView:new()
   self.font = "font"
   self.size.y = 0
   self.label = ""
+  self.mouse_position = {x = 0, y = 0}
```

Comment:
Is this needed?
Instead of saving mouse positions, you could save the hovered entry itself.

Reply:
This is used to know if the mouse is on top of the suggestions box, wouldn't be needed when FloatingView is ready.